site/app/templates/admin/admin_gradeable/AdminGradeableAuto.twig

```diff
@@ -101,7 +101,7 @@
     <div class="notebook-builder-info">
         <div class="option-title">Notebook Builder:</div>
         <div>
-            <a class="btn btn-primary" href="{{ notebook_builder_url ~ '/new' }}">Start New</a>
+            <a class="btn btn-primary" href="{{ notebook_builder_url ~ '/new' }}" >Start New</a>
```

Comment:
Is this change necessary? If not it'd be better to remove it. After this it looks good to merge.